src/config/ConfigManager.cpp

```diff
@@ -4,21 +4,18 @@
 #include <filesystem>
 
 static Hyprlang::CParseResult handleWallpaper(const char* C, const char* V) {
-    const std::string      COMMAND = C;
-    const std::string      VALUE   = V;
-    Hyprlang::CParseResult result;
+    const std::string           COMMAND = C;
+    const std::string           VALUE   = V;
+    Hyprlang::CParseResult      result;
+    Hyprutils::String::CVarList args(VALUE, 3, ',', true);
```

Comment:
add a `using namepsace Hyprutils::String` at the top to avoid this

also this can be a CConstVarList because you arent editing anything